lib/tasks/data_import/custom_fields.rake

```diff
@@ -0,0 +1,43 @@
+namespace :custom_fields do
+  desc 'import custom field from csv file to database'
+  task :build_initial => :environment do
+    file_path = Rails.root.join('lib', 'seeds', 'custom_fields.csv')
+    
+    file = File.open(file_path, "r:ISO-8859-1")
+    csv_parser = CSV.parse(file, headers: true)
+    csv_parser.each.with_index do |row, index|
+      if row['import_id_referece'] == nil then
+        custom_field_section = CustomFieldSection.new(
+          name: row["name"],
+          hint: row["description"],
+          public: row["public"],
+          api_public: row["api_public"],
+          sort_order: row["sort_order"]
+        )
+
+        unless custom_field_section.save
+          puts "CustomFieldSection #{custom_field_section.name} failed to save: #{custom_field_section.errors.messages}"
+        end
+      end
+
+      custom_field_array = csv_parser.select{|item| item['import_id_referece'] == row['import_id']} #searching by reference id and gets bunch of items based on condition
+
+      if (!custom_field_array.empty?) then #if array is not empty
+        custom_field_array.each { |row|
+          custom_field = CustomField.new(
+            key: row["key"],
+            field_type: row["field_type"],
```

Comment:
I don't know how accurately the checks are done behind the scenes for this but but would be good to have checks to make sure `field_type` is an existing custom_field type. 

```
  def self.types
    [
      "Text",
      "Number",
      "Checkbox",
      "Select",
      "Date"
    ]
  end
```

Reply:
added validation for  field_type, it can't be blank